include/quda.h

```diff
@@ -448,6 +448,17 @@ extern "C" {
     /** Whether to use fused kernels for mobius */
     QudaBoolean use_mobius_fused_kernel;
 
+    /**
+     * Parameters for distance preconditioning algorithm proposed in arXiv:1006.4028,
+     * which is useful to solve a precise heavy quark propagator.
+     * alpha0 and t0 follow Eq.(9) in the article.
+     */
+
+    /** The alpha0 parameter for distance preconditioning, related to the pseudoscalar meson mass */
+    double distance_pc_alpha0;
```

Comment:
Both of these parameters need to be replicated in the Fortran interface modules (lib/quda_fortran.F90).  Can you add them there?

Reply:
Done in https://github.com/lattice/quda/pull/1428/commits/afcdc8e04fd24f6f6ca816c37a55f2721317445e